nativelib/src/main/scala/scala/scalanative/runtime/ClassInstancesRegistry.scala

```diff
@@ -0,0 +1,35 @@
+package scala.scalanative.runtime
+
+/** Registry for created instances of java.lang.Class
+ * Its only purpose is to prevent the GC from collecting instances of java.lang.Class
+ **/
```

Comment:
```suggestion
/** Registry for created instances of java.lang.Class
 *  Its only purpose is to prevent the GC from collecting instances of java.lang.Class
 */
```